docs/examples/frameworks/pytorch/pytorch-lightning.ipynb

```diff
@@ -33,7 +33,13 @@
     "\n",
     "import os\n",
     "\n",
-    "BATCH_SIZE = 64"
+    "BATCH_SIZE = 64\n",
+    "\n",
+    "# workaround for https://github.com/pytorch/vision/issues/1938 - error 403 when downloading mnist dataset\n",
+    "from six.moves import urllib\n",
+    "opener = urllib.request.build_opener()\n",
+    "opener.addheaders = [('User-agent', 'Mozilla/5.0')]\n",
+    "urllib.request.install_opener(opener)"
```

Comment:
Does this work with just the urllib.request? https://docs.python.org/3/library/urllib.request.html#module-urllib.request

`six` is compat layer for Python 2.

Reply:
Fixed.